src/Esfa.Vacancy.Register.UnitTests/SearchApprenticeship/Application/GivenASearchApprenticeshipVacanciesRequestValidator/GivenSearchApprenticeshipVacanciesRequestValidatorBase.cs

```diff
@@ -9,10 +9,10 @@ namespace Esfa.Vacancy.Register.UnitTests.SearchApprenticeship.Application.Given
 {
     public abstract class GivenSearchApprenticeshipVacanciesRequestValidatorBase
     {
-        internal Mock<IFrameworkCodeRepository> FrameworkCodeRepositoryMock;
-        internal static List<string> ValidFrameworkCodes = new List<string> { "1" };
-        internal Mock<IStandardRepository> StandardRepositoryMock;
-        internal static List<string> ValidStandardCodes = new List<string>() { "1" };
+        private Mock<IFrameworkCodeRepository> FrameworkCodeRepositoryMock;
+        internal static readonly List<string> ValidFrameworkCodes = new List<string> { "1" };
+        private Mock<IStandardRepository> StandardRepositoryMock;
+        internal static readonly List<string> ValidStandardCodes = new List<string>() { "1" };
```

Comment:
Can this be declared as a readonly property instead? Fields should begin with _ and be lower camel case.